package.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "supports-esm",
+  "version": "1.0.0",
```

```suggestion
  "version": "0.0.0",
```

Should we start at 0.0.0 so we can bump the major in the original release? (So we can still use `npm version`